package.json

```diff
@@ -56,7 +56,7 @@
     "build:prod-client": "node build.js --prod-client",
     "build:prod-server": "node build.js --prod-server",
     "lint-fix": "eslint . --fix",
-    "test": "NODE_ENV=testing jest --config jest.config.json -u",
+    "test": "NODE_ENV=test jest --config jest.config.json -u",
```

Comment:
No need to change, jest default env is testing

Reply:
ok